phpstan-baseline.neon

```diff
@@ -5,33 +5,18 @@ parameters:
 			count: 1
 			path: docs/pages/code/operations/since.php
 
-		-
-			message: "#^Method loophp\\\\collection\\\\Collection\\:\\:all\\(\\) should return array\\<TKey, T\\> but returns array\\.$#"
-			count: 1
-			path: src/Collection.php
-
 		-
 			message: "#^Method loophp\\\\collection\\\\Collection\\:\\:empty\\(\\) return type with generic interface loophp\\\\collection\\\\Contract\\\\Collection does not specify its types\\: TKey, T$#"
 			count: 1
 			path: src/Collection.php
 
-		-
-			message: "#^Method loophp\\\\collection\\\\Collection\\:\\:fromCallable\\(\\) return type with generic class loophp\\\\collection\\\\Collection does not specify its types\\: TKey, T$#"
-			count: 1
-			path: src/Collection.php
-
 		-
 			message: "#^Method loophp\\\\collection\\\\Collection\\:\\:fromIterable\\(\\) return type with generic class loophp\\\\collection\\\\Collection does not specify its types\\: TKey, T$#"
 			count: 1
 			path: src/Collection.php
 
 		-
-			message: "#^Method loophp\\\\collection\\\\Collection\\:\\:getIterator\\(\\) return type has no value type specified in iterable type Iterator\\.$#"
-			count: 1
-			path: src/Collection.php
-
-		-
-			message: "#^Return type \\(Iterator\\) of method loophp\\\\collection\\\\Collection\\:\\:getIterator\\(\\) should be covariant with return type \\(Iterator\\<mixed, mixed\\>\\) of method loophp\\\\collection\\\\Contract\\\\Collection\\<TKey,T\\>\\:\\:getIterator\\(\\)$#"
+			message: "#^Parameter \\#1 \\$callable of class loophp\\\\collection\\\\Collection constructor expects callable\\(\\.\\.\\.mixed\\)\\: iterable\\<int, string\\>, Closure\\(string, string\\)\\: loophp\\\\collection\\\\Iterator\\\\StringIterator\\<mixed, string\\> given\\.$#"
```

Comment:
This seems to be an issue with PHPStan, I've reported it here: https://github.com/phpstan/phpstan/issues/5060